scripts/tests/scripts_regression_tests.py

```diff
@@ -2336,7 +2336,7 @@ def simple_test(self, manual_timing=False):
             provenance_dirs = glob.glob(os.path.join(timing_dir, "performance_archive", getpass.getuser(), casename, lids[0] + "*"))
             self.assertEqual(len(provenance_dirs), 1, msg="provenance dirs were missing")
 
-        self.verify_perms(self, timing_dir)
+        verify_perms(self, timing_dir)
```

Comment:
This was to avoid a pylint error?  Another way to do it is with the 
@staticmethod

Reply:
No, multiple test classes (both BlessTests and SaveTimings) needed this capability, so I made it a free function.